### PR TITLE
added the large test example

### PR DESCRIPTION
**Pull request by candysmurf**
Summary of changes:
-Added the large test example

How to verify it:
- tested it with Minikube.

Testing done:
- unit test
- integration test
- large test.

🐢

### CODE REVIEW

File: scripts/large-compose.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```
Comment (kindermoumoute) [inline scripts/large-compose.sh]: Rename to `large_compose.sh` to make it works with the makefile.

**Comment by kindermoumoute** [comment]
`make test-large` and the example will fail every other collecting event
>ID     					 NAME  						 STATE 		 HIT   	 MISS  	 FAIL  	 CREATED       		 LAST FAILURE
e428be7b-2951-45ad-852d-908b60d0a9a3   	 Task-e428be7b-2951-45ad-852d-908b60d0a9a3     	 Disabled      	 10    	 0     	 10    	 4:47PM 9-20-2016      	 connection is shut down

But `TEST_K8S=1 make test-large` works fine.

File: examples/tasks/elasticsearch-task.json [hunk 2]
```diff
@@ -14,7 +14,12 @@
                 "/intel/elasticsearch/node/*/os/mem/total_in_bytes": {},
                 "/intel/elasticsearch/cluster/status": {}
             },
-            "config": {},
+            "config": {
+                 "/intel/elasticsearch/node": {
+                    "server": "127.0.0.1",
```
Comment (kindermoumoute) [inline examples/tasks/elasticsearch-task.json]: Why don't use the container DNS? Does it work on docker machine?

File: examples/tasks/example.sh [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+# ensure PLUGIN_PATH is set
+TMPDIR=${TMPDIR:-"/tmp"}
+PLUGIN_PATH=${PLUGIN_PATH:-"${TMPDIR}/snap/plugins"}
+mkdir -p $PLUGIN_PATH
+
+
+_info "PLUGIN_PATH=" $PLUGIN_PATH
+
+_info "Get latest plugins"
+(cd $PLUGIN_PATH && curl -sSO http://snap.ci.snap-telemetry.io/plugins/snap-plugin-publisher-file/master/latest/linux/x86_64/snap-plugin-publisher-file && chmod 755 snap-plugin-publisher-file)
```
Comment (kindermoumoute) [inline examples/tasks/example.sh]: Update the flags to `curl -sfLSO ...`. It handles error if the binary is not correct.

**Comment by candysmurf** [comment]
@kindermoumoute, apparently I need to use dynamic IP for local and remote(Kubernetes).  Please review the updated changes.

**Pull request merged**